### PR TITLE
Philipp add predictions utilities

### PR DESCRIPTION
**Pull request by philippmwirth**
# add predictions utilities

Adds utilities to download predictions from a configured datasource, namely:
- `download_prediction_task_names`
- `download_prediction_schema`
- `download_raw_predictions`

### CODE REVIEW

File: tests/api_workflow/test_api_workflow_datasources.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from multiprocessing.sharedctypes import Value
```
Comment (MalteEbner) [inline tests/api_workflow/test_api_workflow_datasources.py:1]: I am not sure why this is needed.

**Review by MalteEbner — COMMENTED** [review]
The functions make sense to me, well done! 
With some unittests for them I would approve.

File: tests/api_workflow/test_api_workflow_datasources.py (hunk 1 above)
Comment (philippmwirth) [reply tests/api_workflow/test_api_workflow_datasources.py:1]: Ugh this was automatically imported for some reason.

**Comment by philippmwirth** [comment]
Will close this one and open up a new one to merge in the feature branch

**Pull request closed without merging**